common/k8stest/util_rdma.go

```diff
@@ -369,3 +373,27 @@ func RestoreConfiguredDisabledRdmaDevicesOnAllMayastorNodes() error {
 	}
 	return nil
 }
+
+// IsVolumeConnectedOverSpecifiedProtocol return true if volume is connected over specified protocol
+// by checking nvme list subsystem live path protocol for specified volume
+func IsVolumeConnectedOverSpecifiedProtocol(volUuid, initiatorNode, protocol string) (bool, error) {
+	// get node IP
+	nodeIp, err := GetNodeIPAddress(initiatorNode)
+	if err != nil {
+		return false, fmt.Errorf("failed to get node IP for node %s, error: %v", initiatorNode, err)
+	} else if *nodeIp == "" {
+		return false, fmt.Errorf("node IP for node %s not found", initiatorNode)
+	}
+	logf.Log.Info("Node IP", "node", initiatorNode, "IP", *nodeIp)
+
+	//volume should get connected over tcp
```

Comment:
this is a generic function so it can check other than tcp also

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/8946ab26ed584de2564a4286465c87140ed86f44..1f86996dcc6348b591b8094ca66fabd2298f1077)